lib/ruby_lsp/ruby_lsp_rails/code_lens.rb

```diff
@@ -168,6 +169,26 @@ def controller?
         class_name.end_with?("Controller") && superclass_name.end_with?("Controller")
       end
 
+      sig { params(node: Prism::DefNode).void }
+      def add_jump_to_view(node)
+        class_name, _ = T.must(@constant_name_stack.last)
+        action_name = node.name
+        controller_name = class_name.delete_suffix("Controller").gsub(/([a-z])([A-Z])/, "\\1_\\2").downcase
```

Comment:
I don't understand the intention behind the `gsub` here, can you explain?

Reply:
No, I wrote it myself. It's turning the controller name into the directory path.